homeassistant/components/peco/config_flow.py

```diff
@@ -0,0 +1,43 @@
+"""Config flow for PECO Outage Counter integration."""
+from __future__ import annotations
+
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import CONF_COUNTY, COUNTY_LIST, DOMAIN
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_COUNTY): vol.In(COUNTY_LIST),
```

Comment:
The county list with just capital letters is not really user-friendly IMHO.
Instead pass a dictionary with key/value pairs, making the values nicely capitalized for the end-user.